cmake/loki_transform.cmake

```
@@ -225,21 +225,23 @@ function( loki_transform_target )
     endif()
 
     if( NOT _PAR_T_TARGET )
-        ecbuild_critical( "The call to loki_transform_target() doesn't specify the TARGET." )
+        ecbuild_critical( "The call to loki_transform_target() doesn't specify the TARGET(S)." )
```

I would suggest to stick to the exact name of the option.
```suggestion
        ecbuild_critical( "The call to loki_transform_target() doesn't specify the TARGET." )
```